dropwizard-auth/src/test/java/io/dropwizard/auth/ChainedAuthProviderTest.java

```diff
@@ -1,12 +1,15 @@
 package io.dropwizard.auth;
```

Comment:
Should this test be moved to package `io.dropwizard.auth.chained` to be consistent with the location of `ChainedAuthFilter`?